app/assets/javascripts/users.js

```diff
@@ -29,3 +29,26 @@ function get_select_values(select) {
 
   return result;
 }
+
+function test_mail(item, id) {
+  $(item).addClass("disable-link");
+  $('#test_indicator').show();
+  var data = '{"user_email" : "' +  $("#user_mail").val() + '"}';
+  var param = JSON.parse(data);
+  $.ajax({
+    url: foreman_url("users/" + id + "/test_mail"),
+    type: 'put',
+    data: param,
+    success: function(result, textstatus, xhr) {
+      notify("<p>" + result.message + "</p>", 'success');
+    },
+    error: function (xhr) {
+      var response = xhr.responseText;
+      notify("<p> Error: " + response + "</p>", 'error');
```

Comment:
Looks better now. I'd do something line `Email was not sent successfuly` or similar, because the response will usually be exception name and stuff unreadable to normal user.